packages/dids/src/methods/did-dht.ts

```diff
@@ -1038,13 +1038,14 @@ export class DidDhtDocument {
           publicKey.alg = parsedAlg || KeyTypeToDefaultAlgorithmMap[Number(t) as DidDhtRegisteredKeyType];
 
           // Determine the Key ID (kid): '0' for the identity key or JWK thumbprint for others.
-          publicKey.kid = dnsRecordId.endsWith('0') ? '0' : await computeJwkThumbprint({ jwk: publicKey });
+          const kid = dnsRecordId.endsWith('0') ? '0' : await computeJwkThumbprint({ jwk: publicKey });
+          publicKey.kid = kid;
 
           // Initialize the `verificationMethod` array if it does not already exist.
           didDocument.verificationMethod ??= [];
 
           // Prepend the DID URI to the ID fragment to form the full verification method ID.
```

Comment:
update comment

Reply:
What would you like me to update the comment to? I'll do it!